architectures/BDR-Always-ON/commands/upgrade.yml

```diff
@@ -82,7 +87,7 @@
   become_user: root
   become: yes
   hosts: "{{ update_hosts|default('all') }}"
-  serial: 1
+  serial: "{{ update_hosts|ternary('1',omit) }}" 
```

Comment:
I don't know what the intention was here, since there is no comment and the commit that introduced it says nothing more than "BDR/PGD code updates", but this change is unacceptable.

Reply:
I agree and can remove it.

Reply:
Removed in `4386b25`